composer/loggers/wandb_logger.py

```diff
@@ -335,4 +354,36 @@ def _convert_to_wandb_image(image: Union[np.ndarray, torch.Tensor], channels_las
     if not channels_last:
```

Comment:
This section is causing a test to fail (and causes the image visualizer to fail when i run it locally).

Reply:
fixed?